Cargo.toml

```diff
@@ -95,12 +96,14 @@ new-e2e-tests = []
 members = [
```

Comment:
Somehow `publish` wasn't in here. Not sure how that was working before... :shrug: